base/array.jl

```diff
@@ -420,6 +420,11 @@ julia> collect(Float64, 1:2:5)
  1.0
  3.0
  5.0
+
+julia> collect(Dict(1 => 2, 2 => "foo"))
```

Comment:
should it be somewhere else? I don't see an exact match for this doctest anywhere, unless someone else deleted it?